x/model/handler_test.go

```diff
@@ -727,7 +730,7 @@ func TestHandler_UpdateModelVersion(t *testing.T) {
 	require.Equal(t, receivedModelVersion.SoftwareVersion, msgUpdateModelVersion.SoftwareVersion)
 
 	require.Equal(t, receivedModelVersion.SoftwareVersionValid, msgUpdateModelVersion.SoftwareVersionValid)
-	require.Equal(t, receivedModelVersion.OtaUrl, msgUpdateModelVersion.OtaUrl)
+	require.Equal(t, receivedModelVersion.OtaUrl, msgCreateModelVersion.OtaUrl)
```

Comment:
Please check that other OTA fields are also not updated

Reply:
Checked